codegen/smithy-dafny-codegen-test/src/test/java/software/amazon/polymorph/smithyjava/JavaTestModels.java

```diff
@@ -20,41 +20,61 @@ class JavaTestModels extends TestModelTest {
-    DISABLED_TESTS.add("Dependencies");
-    DISABLED_TESTS.add("Extern");
-    DISABLED_TESTS.add("LanguageSpecificLogic");
-    DISABLED_TESTS.add("Refinement");
-    DISABLED_TESTS.add("SimpleTypes/BigDecimal");
-    DISABLED_TESTS.add("SimpleTypes/BigInteger");
-    DISABLED_TESTS.add("SimpleTypes/SimpleBlob");
-    DISABLED_TESTS.add("SimpleTypes/SimpleBoolean");
-    DISABLED_TESTS.add("SimpleTypes/SimpleByte");
-    DISABLED_TESTS.add("SimpleTypes/SimpleDocument");
-    DISABLED_TESTS.add("SimpleTypes/SimpleDouble");
-    DISABLED_TESTS.add("SimpleTypes/SimpleEnum");
-    DISABLED_TESTS.add("SimpleTypes/SimpleEnumV2");
-    DISABLED_TESTS.add("SimpleTypes/SimpleFloat");
-    DISABLED_TESTS.add("SimpleTypes/SimpleInteger");
-    DISABLED_TESTS.add("SimpleTypes/SimpleLong");
-    DISABLED_TESTS.add("SimpleTypes/SimpleShort");
-    DISABLED_TESTS.add("SimpleTypes/SimpleString");
-    DISABLED_TESTS.add("SimpleTypes/SimpleTimestamp");
-    DISABLED_TESTS.add("Streaming");
-    DISABLED_TESTS.add("Union");
-    DISABLED_TESTS.add("aws-sdks/kms-lite");
-    DISABLED_TESTS.add("aws-sdks/sqs");
-    DISABLED_TESTS.add("aws-sdks/sqs-via-cli");
-    //TODO: Add support for Recursive shapes.
-    DISABLED_TESTS.add("RecursiveShape");
-    // S3 is not yet supported
-    DISABLED_TESTS.add("aws-sdks/s3");
+    DISABLED_TESTS.add("Dependencies"); // Smithy-Dafny Error
+    DISABLED_TESTS.add("CallingAWSSDKFromLocalService"); // Different types across type conversion (wrapped vs not)
 
-    //TODO: https://github.com/smithy-lang/smithy-dafny/issues/599
-    DISABLED_TESTS.add("Positional");
+    DISABLED_TESTS.add("LanguageSpecificLogic"); // Smithy-Dafny Error
+    DISABLED_TESTS.add("Positional"); // Smithy-Dafny Error
+    DISABLED_TESTS.add("Refinement"); // Smithy-Dafny Error
+    DISABLED_TESTS.add("SimpleTypes/SimpleBoolean"); // Need runtime
+    DISABLED_TESTS.add("SimpleTypes/SimpleDouble"); // Error in generated type conversion
+    DISABLED_TESTS.add("SimpleTypes/SimpleTimestamp"); // Need to add for Java
+
+    DISABLED_TESTS.add("aws-sdks/kms-lite"); // Not written yet
+    DISABLED_TESTS.add("aws-sdks/s3"); // Not written yet
+    DISABLED_TESTS.add("aws-sdks/sqs"); // Not written yet
+    DISABLED_TESTS.add("aws-sdks/sqs-via-cli"); // Not written yet
+
+    DISABLED_TESTS.add("AggregateReferences"); // Not supported yet
+    DISABLED_TESTS.add("RecursiveShape"); // Not supported yet
+    DISABLED_TESTS.add("SimpleTypes/BigDecimal"); // Not supported yet
+    DISABLED_TESTS.add("SimpleTypes/BigInteger"); // Not supported yet
+    DISABLED_TESTS.add("SimpleTypes/SimpleByte"); // Not supported yet
+    DISABLED_TESTS.add("SimpleTypes/SimpleDocument"); // Not supported yet
+    DISABLED_TESTS.add("SimpleTypes/SimpleFloat"); // Not supported yet
+    DISABLED_TESTS.add("SimpleTypes/SimpleShort"); // Not supported yet
+    DISABLED_TESTS.add("Streaming"); // Not supported yet
+    //    These are commented out because they should work
```

Comment:
For the record I don't love this, since it will fall out of date, but won't block on it. I've been real close to adding an option/environment variable for running a single test multiple times before. :)

Reply:
I don't either, and it will get out of date. It sucks, but having the list sucks less for me right this second as I worked on these tests. A way to say "just this test" would be wonderful!